demos/tutorial/alphabets/example_ordValue.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <seqan/basic.h>
+
+using namespace seqan;
+
+int main(){
+
+//![main]
+Dna a = 'A';
+Dna c = 'C';
```

Comment:
The Tutorial Webpages often do not include the main function, thus the (correct) indentation will occur "without a reason" for the person who is just looking at the webpage and not the file. 
But I assume a consistent code style is more important than a slight change of the look on the Webpage?